db/fixtures/lessons/react_lessons.rb

```diff
@@ -213,12 +206,19 @@ def react_lessons
       previewable: true,
       identifier_uuid: '8e0206b5-e189-4ec3-b58b-c7f0afb84570',
     },
-    'Conclusion' => {
+    'Conclusion (Node Path)' => {
+      title: 'Conclusion',
+      description: 'Conclusion',
+      is_project: false,
+      github_path: '/react/conclusion/conclusion_full_stack_javascript.md',
+      identifier_uuid: 'create_uuid',
+    },
+    'Conclusion (Rails Path)' => {
       title: 'Conclusion',
       description: 'Conclusion',
       is_project: false,
-      github_path: '/react/conclusion/conclusion.md',
-      identifier_uuid: '8s7d6f5d-4d3d-2d1d-1d2d-3d4d5d6d7d8d',
+      github_path: '/react/conclusion/conclusion_ruby_on_rails.md',
+      identifier_uuid: 'create_uuid',
```

Comment:
We need this to be a uuid please. Running the seeds task will do it automatically but you can grab a random uuid from any [uuid generator](https://www.uuidgenerator.net/) if you can't get your db working.

Reply:
Done, is the naming i used for lessons correct, i.e, Conclusion (Rails Path)
while the title property inside is still "Conclusion"

Reply:
I'd need to get on any a review app once CI is passing to be sure, but it looks good to me at a code level.

Reply:
Committed it